libbeat/tests/system/test_meta.py

```diff
@@ -0,0 +1,36 @@
+from base import BaseTest
+
+import os
+import stat
+
+
+INTEGRATION_TESTS = os.environ.get('INTEGRATION_TESTS', False)
```

Comment:
what is this variable about?

Reply:
It's doing nothing currently 😝. I've fixed this in d370d5854. Thanks for catching!